tests/unit/test_logger.py

```diff
@@ -117,7 +115,7 @@ def test_nested_exceptions(self, caplog):
                 "{red}ERROR{nc}: message - second: first\n"
                 "{red}{line}{nc}\n"
                 "{stack_trace}"
-                "{red}{line}{nc}\n".format(
```

Comment:
Could you please explain why `\n` are getting removed now?

Reply:
I found it excessive, maybe not enough reason to do so.

Reply:
@mroutis The newline is needed there, it is part of the message. I imagine you are trying to solve double newline between the exc and "Having any troubles" footer, right? If so, it is probably worth just removing \n there. But again, this is out of scope for this PR, so let's not touch new lines at all, please roll back this change for now.

Reply:
Ok